kratos/utilities/nurbs_utilities/projection_nurbs_geometry_utilities.h

```diff
@@ -124,15 +124,15 @@ namespace Kratos
         const double Accuracy = 1e-6)
     {
         // Initialize variables
-        bool clampU = false;
-        bool clampV = false;
+        bool conditionFirstRowZero, conditionSecondRowZero, conditionFirstColumnZero, conditionSecondColumnZero, isSystemInvertible;
```

Comment:
I see that some of them are already there but the names should be lower case with underscores
Can you please update?

Reply:
Any ideas why travis is still failing? It seems that it fails in some other tests .. Thanks!

Reply:
`TestNurbsSurfaceGeometryProjection3d` fails

Reply:
Right, got to fix that, danke!